### ISSUE TITLE
Montaż gum do ściany

### ISSUE THREAD

**Issue by MaciejSm** [zhalat/Flat#2]
Witam, jak montowałeś gumy do ściany? Czy były klejone do ściany? Jeśli tak, to mogę wiedzieć na co? Widzę też, że dodatkowo były przytwierdzane do ściany jakimiś białymi krążkami. Co to dokładnie jest? Zastanawiam się czy sam klej by nie wystarczył? Czy takie nawiercanie ściany, żeby przymocować gumę nie obniży jej izolacji akustycznej?

**Comment by zhalat** [comment]
cześć,
Na początku na kelj + kołki do steropianu, potem juz tylko na kelj. Ważne jest dobrze przygotowane podłoze. Bez farby i zagruntowane. Zapewne dziurawienie nie pomaga ale obawialem się o to czy kelj bedzie trzymal po latach

**Comment by zhalat** [comment]
Tak, po tym jak zacząłem kleić na dobrze przygotowane podłoże kołków juz nie dawałem.
Kleiłem na klej z castoramy - klej montażowy.  Jenak wiem że ten klej nie jest wysoko jakościowy. Jednak  ze wzgledu na cene używałem go bo schodzilo tego sporo.

**Comment by MaciejSm** [comment]
Dobrze przygotowane podłoże to zagruntowanie? Czy zastosowałeś coś jeszcze? No i ostatnie dwa pytania — stelarz trzymający wełnę został zamontowany bezpośrednio przy gumie, dotyka jej, czy zastosowałeś jakiś odstęp? Czy karton-gips dochodzi do samej ściany, czy jest jakaś dylatacja? Wielkie dzięki za ten poradnik i ogromne dzięki za Twoje odpowiedzi na moje pytania.

**Comment by zhalat** [comment]
Zagruntowane i zdarta farba (chybe ze się dobrze trzyma). Stelaż łączy sie z sufitem i podlogą za poprzez gume 
cos takiego https://www.castorama.pl/mata-antywibracyjna-diall-60-x-60-x-1-5-cm-czarna-id-1066639.html
tyle że kupoawelm duzo taniej. G-K nie dotyka ścian- jest mala dylatacja zaklejna na koncu akrylem
